bibliopixel/remote/trigger_process.py

```diff
@@ -1,9 +1,9 @@
 from .. util import importer
 
 
-def run_trigger(typename, q, configs):
+def run_trigger(typename, q, events, kwargs):
```

Comment:
Ahh, ok... that's really just an artifact of rapidly developing this... when I started I only needed `q` and `configs` so `kwargs` was overkill. But then I added that... and I could've then just simplified the whole thing, but I didn't because it worked. I have a love-hate relationship with kwargs... it's makes the code way less verbose but also makes it super hard to follow sometime. I've had plenty of long sessions trying to figure out what the hell code you wrote is doing exactly because parameters just get spirited away into kwargs until they are used later :P